source/patterns/@aws-solutions-constructs/aws-lambda-kinesisstreams/test/lambda/index.mjs

```diff
@@ -0,0 +1,15 @@
+import { KinesisClient, PutRecordCommand } from "@aws-sdk/client-kinesis";
```

Comment:
.mjs?

Reply:
Node 18 runtime supports es6 module files (the .mjs extension). Since the test lambda is node18, it made sense to use the mjs file here to follow conventions.